docs/topics/ref_rhoardoc-jira-fields.adoc

```diff
@@ -0,0 +1,61 @@
+
+[id='rhoardoc-jira-fields_{context}']
+= RHOARDOC JIRA fields
+
+The following fields are used in the link:https://issues.jboss.org/browse/RHOARDOC[RHOARDOC^] JIRA project:
+
+.RHOARDOC JIRA fields
+Issue Type:: The type of the issue.
++
+--
+The following types are used:
+
+* *Bug:* An error in the documentation.
+* *Enhancement:* An improvement or a missing part of the documentation.
+* *Epic:* A large requirement.
+Multiple JIRAs that track individual work items are linked to it.
+* *Task:* A task to be done.
+--
+
+Priority:: The importance of the issue.
++
+--
+The following priorities are used:
+
+* *Blocker:* The release of the documentation can not proceed until the issue is fixed.
+* *Critical:* The issue must be resolved quickly, but is not blocking.
+* *Major:* The default importance of an issue.
+* *Minor:* The issue is of low importance can be deferred to a later date.
+--
+
+Summary:: A brief description of the issue.
++
+This is the displayed name of the issue.
+Include important information about the issue like its nature and where it occurs, but be brief.
+
+Description:: A complete description of the issue.
+
+Sprint:: Which sprint the issue belongs to.
++
+Only put an issue in a sprint if you are reasonably confident you can finish it within the sprint.
+
+Story Points:: The amount of effort the issue requires.
++
+Numbers from the Fibonacci sequence are used, each denoting an increasing workload: *1, 2, 3, 5, 8, 13.*
```

Comment:
Maybe we could add a note here that the recommended number of total story points per writer per sprint is 13. Or are we not set on that yet? WDYT?

Reply:
So if I get that right, when I create a JIRA issue with 13 story points for example, then that issue would span the whole sprint? :)

Reply:
@inoxx03 Sure, I'll add it.

@spisiakm That's the recommendation, yes. Anything larger is probably an epic and should be broken down into smaller JIRAs.

Reply:
@spisiakm Yes, that might happen if the issue sin question is truly monolithic. Or, if it isn't, the issue might be split into smaller chunks that can be done over the course of several sprints.
Also, the 13 story points per sprint are recommended not prescribed, so more points are acceptable, if the writer can handle it.

Reply:
@tradej @inoxx03 Thanks for clarification!